patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -45,7 +45,7 @@ index e5a9fd2d98b65ff7bb54ded343d1de3891b232a8..70a15ac085e9c2dd413d73b13d252dfe
  
 +<!--
      <uses-permission android:name="com.google.android.apps.now.CURRENT_ACCOUNT_ACCESS" />
-+-->
++--> <uses-permission android:name="com.google.android.finsky.permission.BIND_GET_INSTALL_REFERRER_SERVICE"/>
```

Comment:
This is for a quick fix as we need it ASAP, follow up issue for proper AndroidManifest patching is created https://github.com/brave/brave-browser/issues/15942